jackson/src/test/java/feign/jackson/JacksonCodecTest.java

```diff
@@ -97,6 +97,15 @@ public void nullBodyDecodesToNull() throws Exception {
   }
 
   @Test
+  public void emptyBodyDecodesToNull() throws Exception {
+    Response
+        response =
+        Response
+            .create(204, "OK", Collections.<String, Collection<String>>emptyMap(), "".getBytes());
```

Comment:
If I were reviewing the code, I probably would have had the same comment. Thanks.